lua/weapons/weapon_squee/shared.lua

```diff
@@ -15,3 +15,7 @@ end
 function SWEP:SecondaryAttack()
 	self:ExtEmitSound("boop.wav", {shared=true})
 end
+
+function SWEP:Reload()
```

Comment:
I'm pretty sure :Reload() fires as full auto while its held down, so this needs some extra logic (look at another soundgun with a reload sound)

Reply:
Yep, forgot to add that. Added

Reply:
Delay is too short

Reply:
I increased it. Now it has time for the full sound to play plus a small buffer